### PR TITLE
ClassName for material slate and Material editable to accept all props

### PR DESCRIPTION
**Pull request by sureshsevarthi**

### CODE REVIEW

File: src/slate-react/MaterialSlate.jsx [hunk 1]
```diff
@@ -27,7 +27,10 @@ export default function MaterialSlate({
   const classes = useStyles()
   return (
     <Slate value={value} editor={editor} onChange={value => onChange(value)}>
-      <Box className={classes.root} borderRadius="borderRadius">
+      <Box
+        className={`${className} ${classes.root}`}
```
Comment (merlos) [inline src/slate-react/MaterialSlate.jsx:31]: It should be the opposite. if not, classes.root will overwrite whatever the className sets...

Comment (sureshsevarthi) [reply src/slate-react/MaterialSlate.jsx:31]: @merlos Yes, it only has border, if they want they can override the class. Don't we allow them to do it?

Comment (merlos) [reply src/slate-react/MaterialSlate.jsx:31]: the first class (className) will be overwritten by the second class (classes.root) if I want to have a blue border, I won't be able. 
Please change the order

Comment (sureshsevarthi) [reply src/slate-react/MaterialSlate.jsx]: yes, your correct. I thought the other way .

**Comment by merlos** [comment]
PR included in #8

**Pull request closed without merging**